tests/queries/0_stateless/02813_seriesOutliersTukey.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS tb1;
```

Comment:
Let's rename this file and the reference file to `02813_seriesOutliersDetectTukey.sql`/`.reference`